test/polytopes.jl

```diff
@@ -264,6 +264,21 @@ end
   r = Ring(merc.([(0, 0), (1, 0), (1, 1), (0, 1)]))
   @test crs(centroid(r)) === crs(r)
 
+  # parameterization
+  ri = Ring(latlon(45, 0), latlon(45, 90), latlon(90, 90))
+  @test ri(T(0)) == latlon(45, 0)
+  @test ri(T(0.25)) ≈ latlon(45, 56.25) atol = 1e7 * eps(T) * u"m"
+  @test ri(T(0.5)) ≈ latlon(60, 90) atol = 1e7 * eps(T) * u"m"
+  @test ri(T(0.75)) ≈ latlon(82.5, 75) atol = 1e7 * eps(T) * u"m"
+  @test ri(T(1)) == latlon(45, 0)
+
+  ro = Rope(latlon(45, 0), latlon(45, 90), latlon(90, 90))
+  @test ro(T(0)) == latlon(45, 0)
+  @test ro(T(0.25)) ≈ latlon(45, 39.375) atol = 1e7 * eps(T) * u"m"
+  @test ro(T(0.5)) ≈ latlon(45, 78.75) atol = 1e7 * eps(T) * u"m"
+  @test ro(T(0.75)) ≈ latlon(63.75, 90) atol = 1e7 * eps(T) * u"m"
+  @test ro(T(1)) == latlon(90, 90)
```

Comment:
Approximate comparison with `LatLon` coordinates can be challenging because it is based on a fallback method that converts to `Cartesian` coordinates over the surface of the Earth. A tiny change in latitude/longitude degrees leads to a large distance in meters.

Our `measure` of geodesic `Segment`, i.e., a curved segment over the Earth surface, is not accurate at the moment. For now, I think we can only test the values t=0 and t=1 safely.

Reply:
I removed the approximate `LatLon` tests in [bcaf0f2](https://github.com/JuliaGeometry/Meshes.jl/pull/1155/commits/bcaf0f261475c278e91e8602d77ee761b4386766).